ember-scoped-css/src/lib/scoped-css-preprocessor.js

```diff
@@ -196,17 +217,28 @@ export default class ScopedCssPreprocessor {
   }
 
   toTree(inputNode, inputPath, outputDirectory, options) {
-    const otherTrees = this.preprocessors.map((p) => p.toTree(...arguments));
+    const otherPreprocessors = this.preprocessors.filter((p) => p !== this);
+    const otherTrees = otherPreprocessors.map((p) => p.toTree(...arguments));
     let mergedOtherTrees = new MergeTrees(otherTrees);
 
-    let componentsNode = new Funnel(inputNode, {
-      include: [
-        this.appName + '/components/**/*.css',
-        ...COMPONENT_EXTENSIONS.map(
-          (ext) => this.appName + `/components/**/*.${ext}`,
-        ),
-      ],
-    });
+    let roots = [
+      this.appName + '/components/',
+      ...(this.userOptions.additionalRoots || []).map(
```

Comment:
this ended up being the primary motivator for the `additionalRoots` option -- we had a few places that hardcoded to searching for components.

atm, `additionalRoots` is only needed for classic builds.